homeassistant/components/switchbot_cloud/light.py

```diff
@@ -149,11 +150,36 @@ async def _send_rgb_color_command(self, rgb_color: tuple) -> None:
         )
 
 
+class SwitchBotCloudCeilingLight(SwitchBotCloudLight):
+    """Representation of SwitchBot Ceiling Light."""
+
+    _attr_max_color_temp_kelvin = 6500
+    _attr_min_color_temp_kelvin = 2700
+
+    _attr_supported_color_modes = {ColorMode.COLOR_TEMP}
+
+    async def _send_brightness_command(self, brightness: int) -> None:
+        """Send a brightness command."""
+        await self.send_api_command(
+            CeilingLightCommands.SET_BRIGHTNESS,
+            parameters=str(value_map_brightness(brightness)),
+        )
+
+    async def _send_color_temperature_command(self, color_temp_kelvin: int) -> None:
+        """Send a color temperature command."""
+        await self.send_api_command(
+            CeilingLightCommands.SET_COLOR_TEMPERATURE,
```

Comment:
These, arguably, could be the same commands as `RGBWWLightCommands`, since they use the same actual commands (here: https://github.com/SeraphicCorp/py-switchbot-api/blob/main/switchbot_api/commands.py#L294C30-L294C49). 

But they're separate on the `py-switchbot-api` side, so I've left them distinct here too. Maybe we should unify that somehow?